snuba/datasets/__init__.py

```diff
@@ -34,6 +45,21 @@ def row_from_processed_message(self, message):
 
         return values
 
+    def get_message_topic(self):
```

Comment:
I think we could consider having a separate class "KafkaDataset" or so that defines the Kafka topics configuration to make the main Dataset more abstract. But it seems a little overkill to me at this point since the dependency of snuba on Kafka is very deep anyway.